docs/synapse/devopsguide.rst

```diff
@@ -231,6 +231,38 @@ environment to help identify any tweaks that may be necessary due to the updated
 
     Please ensure you have a tested backup available before applying these updates.
 
+.. _devops-task-storm-pool:
+
+Configure a Storm Query Pool
```

Comment:
Do we want to move this down to the Cortex specific configuration options ( around lines 920 )

Reply:
I think this task-based step is best suited to devops tasks (or even an optional step in the deployment guide) to promote visibility of what is likely to become an option enabled in most/all production instances :)